sources/web/datalab/polymer/modules/template-manager/template-manager.ts

```diff
@@ -126,6 +127,24 @@ class BigQueryTableOverviewTemplate extends NotebookTemplate {
 }
 
 /**
+ * This template contains intro cells so the notebook isn't empty on creation.
+ */
+class NewNotebookTemplate extends NotebookTemplate {
+  constructor(dict: { [key: string]: any }) {
+    const parameters = Object.keys(dict).map((k) => ({name: k, value: dict[k]}));
+
+    // Specify the default location of the template.
+    const defaultTemplateLocation = 'static/templates/NewNotebookTemplate.ipynb';
+
+    const windowDatalab = window.datalab || {};
```

Comment:
Can we move this to the global scope in `template-manager.ts` and clean it up from all template constructors? It's really meant to be done once, and is there only for development.